tests/foreman/cli/test_repository.py

```diff
@@ -2577,19 +2577,17 @@ def test_positive_sync_publish_promote_cv(self, repo, module_org, module_product
         """
         lce = make_lifecycle_environment({'organization-id': module_org.id})
         Repository.synchronize({'id': repo['id']})
+        synced_repo = Repository.info({'id': repo['id']})
+        assert synced_repo['sync']['status'] == 'Success'
```

Comment:
```suggestion
        assert synced_repo['sync']['status'].lower() == 'success'
```
This would save you from future silly failure in the `success` word.